src/DetectChanges/BCBreak/ClassBased/MethodChanged.php

```diff
@@ -48,10 +45,10 @@ private function methods(ReflectionClass $class): array
     {
         $methods = $class->getMethods();
 
-        return array_combine(
-            array_map(static function (ReflectionMethod $method): string {
-                return strtolower($method->getName());
-            }, $methods),
+        return Dict\associate(
+            Vec\map($methods, static function (ReflectionMethod $method): string {
+                return Str\lowercase($method->getName());
```

Comment:
`strtolower()` worked OK here - why replace it? :thinking:

Reply:
consistency, that's the only reason, as well as guaranteed consistent behavior when it comes to type erorrs ( this https://wiki.php.net/rfc/consistent_type_errors, but for PHP 7.4 )